### PR TITLE
7/1: Step 3 of comments walkthrough

### PR DESCRIPTION
**Pull request by annemariecabs**
Create comments.html page to display comments on. Update DataServlet to create an ArrayList of hard-coded comments then sent those packaged as JSON as the response to fetch(). Write JS method getComments() to fetch that response and display it on the Comments page when it is loaded.

NOTE: My branches migrated to main repo during this PR so the first comments are in the still open #35 PR.

### CODE REVIEW

File: agcaballero/portfolio/src/main/webapp/nav.html [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<ul id="nav-list" class="nav-ul">
+  <li class="nav-li"><a href="index.html">About</a></li>
+  <li class="nav-li"><a href="gallery.html">Gallery</a></li>
+  <li class="nav-li"><a href="blog.html">Blog</a></li>
+  <li class="nav-li" id="last-elem">AnneMarie Caballero</li>
+</ul>
```
Comment (ykjchen) [inline agcaballero/portfolio/src/main/webapp/nav.html:6]: Adding a newline at the end avoids this warning.

Comment (annemariecabs) [reply agcaballero/portfolio/src/main/webapp/nav.html]: Done

File: makefile [hunk 2]
```diff
@@ -11,23 +11,23 @@ pretty: node_modules
...
 	find agcaballero/portfolio/src/main -iname *.java | xargs $(CLANG_FORMAT) -i
 	find agcaballero/portfolio/src/main -iname *.js | xargs $(CLANG_FORMAT) -i
-	find jelares/portfolio/src/main -iname *.html -o -iname *.css | xargs $(PRETTIER) --write
-	find jelares/portfolio/src/main -iname *.java | xargs $(CLANG_FORMAT) -i
-	find jelares/portfolio/src/main -iname *.js | xargs $(CLANG_FORMAT) -i
-	find udaykalra/portfolio/src/main -iname *.html -o -iname *.css | xargs $(PRETTIER) --write
-	find udaykalra/portfolio/src/main -iname *.java | xargs $(CLANG_FORMAT) -i
-	find udaykalra/portfolio/src/main -iname *.js | xargs $(CLANG_FORMAT) -i
+#	find jelares/portfolio/src/main -iname *.html -o -iname *.css | xargs $(PRETTIER) --write
+#	find jelares/portfolio/src/main -iname *.java | xargs $(CLANG_FORMAT) -i
```
Comment (aaronegolden) [inline makefile]: May as well just delete these instead of commenting out.

Comment (annemariecabs) [reply makefile]: I believe I removed this from the commit just because I didn't want it to affect the repo.

Comment (annemariecabs) [reply makefile]: Done

**Pull request merged**